nettests/ndt.go

```diff
@@ -19,73 +19,59 @@ func (n NDT) Run(ctl *Controller) error {
 
 // NDTTestKeys for the test
 type NDTTestKeys struct {
-	Upload     float64 `json:"upload"`
-	Download   float64 `json:"download"`
-	Ping       float64 `json:"ping"`
-	MaxRTT     float64 `json:"max_rtt"`
-	AvgRTT     float64 `json:"avg_rtt"`
-	MinRTT     float64 `json:"min_rtt"`
-	MSS        float64 `json:"mss"`
-	OutOfOrder float64 `json:"out_of_order"`
-	PacketLoss float64 `json:"packet_loss"`
-	Timeouts   float64 `json:"timeouts"`
-	IsAnomaly  bool    `json:"-"`
+	Upload         float64 `json:"upload"`
+	Download       float64 `json:"download"`
+	Ping           float64 `json:"ping"`
+	MaxRTT         float64 `json:"max_rtt"`
+	AvgRTT         float64 `json:"avg_rtt"`
+	MinRTT         float64 `json:"min_rtt"`
+	MSS            float64 `json:"mss"`
+	RetransmitRate float64 `json:"retransmit_rate"`
+	IsAnomaly      bool    `json:"-"`
```

Comment:
cc: @sarathms @hellais 

The above diff summarises what changed in the emitted message migrating from ndt5 to ndt7.

Reply:
RetransmitRate is conceptually similar to PacketLoss, but PacketLoss was too much of a lie to keep the naming.